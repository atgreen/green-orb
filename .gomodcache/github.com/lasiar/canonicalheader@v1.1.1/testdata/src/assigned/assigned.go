package assigned

import (
	"fmt"
	"net/http"
)

func _() {
	h := http.Header{}

	i, g := 0, h.Del
	fmt.Println(i)
	g("TT") // want `non-canonical header "TT", instead use: "Tt"`

	f := h.Get
	f("TT") // want `non-canonical header "TT", instead use: "Tt"`
}
