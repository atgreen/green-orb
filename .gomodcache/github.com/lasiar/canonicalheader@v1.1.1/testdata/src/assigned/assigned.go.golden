package assigned

import (
	"fmt"
	"net/http"
)

func _() {
	h := http.Header{}

	i, g := 0, h.Del
	fmt.Println(i)
	g("Tt") // want `non-canonical header "TT", instead use: "Tt"`

	f := h.Get
	f("Tt") // want `non-canonical header "TT", instead use: "Tt"`
}
